<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DEliminarUsuarios</class>
 <widget class="QDialog" name="DEliminarUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="gvFoto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leId">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lEliminarUsuario">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>ELIMINAR USUARIO</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lId">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce el ID del usuario que quieres eliminar: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
